src/models/Org.js

```diff
@@ -0,0 +1,19 @@
+const fetch = require('isomorphic-fetch');
+const Repo = require('./Repo');
+
+class Org {
+  constructor(org) {
+    this.org = org;
+    this.login = () => this.org.login;
```

Comment:
@michaelmurray6298 does this need to be a function? Could you just do `this.login = this.org.login;` ?